tests/storage/mailbox_test.py

```diff
@@ -357,13 +421,15 @@ def reply_msg_callback(vol_data):
                         # check that SPM inbox was cleared
                         if f.read(sm.MAILBOX_SIZE) == sm.EMPTYMAILBOX:
                             break
-                    assert time.time() < deadline, "Timeout clearing SPM inbox"
-                    time.sleep(0.1)
+                    if time.monotonic() < deadline:
```

Comment:
Whats' wrong with `assert`?

Reply:
>If a timeout expires, raise RuntimeError instead of assert. This is not a test failure but an error.

Reply:
OK :)
Since those are tests, I thought `assert` are appropriate :)

Reply:
assert has 2 issues:
- It confuse the reader that this is part of the test, but this check is not the purpose of the test
- The syntax does not allow long assert message. If you want to add a longer message it is much easier to do this by raising an exception.